win32ss/gdi/ntgdi/line.c

```diff
@@ -533,7 +535,7 @@ NtGdiLineTo(HDC  hDC,
 // FIXME: This function is completely broken
 BOOL
 APIENTRY
```

Comment:
```suggestion
static
BOOL
```
`GdiPolyDraw` is not used outside ot the TU. Some compilers can do more optimization of static functions.

Reply:
Done.